shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -402,12 +403,30 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
                                CGRectGetWidth(flutter_view.bounds),
                                CGRectGetHeight(flutter_view.bounds))] autorelease];
 
+  NSMutableArray* blurRadii =
+      [[NSMutableArray alloc] init];  // TODO EMILY: is autorelease the best option?
```

Comment:
Let's resolve this TODO.

Reply:
Since this object is used in a different function, autorelease is probably fine for simplicity.